framework/include/araui/ui_animation.h

```diff
@@ -42,14 +50,102 @@ typedef enum {
 extern "C" {
 #endif
 
+/**
+ * @brief Create a move animation
+ *
+ * If you want to get a current coordinate of widget, use ui_widget_get_rect.
+ *
+ * @param[in] from_x The x coordinate to move from
```

Comment:
You don't have to add `The` when the statement is started.

Reply:
Fixed